F-Cabinet/index.md

```diff
@@ -115,13 +115,12 @@ What does this mean for us in light of the fact that our kernel config specifies
 CONFIG_PGTABLE_LEVELS=4
 ```
 
-If we look at the [sample
-session](https://cs4118.github.io/cabinet/testing/session1.html)
-from the Cabinet prompt, it shows that the `p4d_paddr` and `pud_paddr` are
-identical.
+If we look at the output from an example program that reports the physical
+addresses of the process' page frame and of the intermediate page tables,
+it shows that the `p4d_paddr` and `pud_paddr` are identical.
 
 ```
-[405] inspect_cabinet():
+[405] inspect_physical_address():
```

Comment:
wasn't really sure how to edit this part since the sessions for farfetchd are different